compiler/src/iree/compiler/Dialect/HAL/Transforms/MaterializeInterfaces.cpp

```diff
@@ -303,10 +305,26 @@ convertBindingUsage(mlir::FunctionOpInterface sourceFuncOp, BlockArgument arg,
     OpBuilder builder(oldOp);
     auto alignmentAttr = sourceFuncOp.getArgAttrOfType<IntegerAttr>(
         arg.getArgNumber(), "stream.alignment");
+
+    constexpr llvm::StringRef subSpanAccessAttr = "iree.codegen.access";
```

Comment:
We might want to do put this as a StringRef in the dialect definition or use the whole dialect attrs thing so it's not a constexpr in a random file.